west.yml

```diff
@@ -150,8 +150,8 @@ manifest:
       groups:
         - hal
     - name: liblc3
-      revision: 54c047249bc6b8089e85c1433b5aa6001bbc1ba9
-      path: modules/lib/liblc3codec
+      revision: pull/1/head
```

Comment:
Is this intentional? Wouldn't that cause us to use the newest always, instead of a specific revision?

Reply:
This is still DNM

Reply:
No, this is the standard way to submit module updates:
https://docs.zephyrproject.org/latest/develop/modules.html#submitting-new-modules

But I now realized that the module.yml is actually upstream, so submitted:
https://github.com/google/liblc3/pull/8